dot/parachain/backing/candidate_backing.go

```diff
@@ -113,19 +116,20 @@ type attestingData struct {
 // tableContext represents the contextual information associated with a validator and groups
 // for a table under a relay-parent.
 type tableContext struct {
-	validator  *validator
-	groups     map[parachaintypes.ParaID][]parachaintypes.ValidatorIndex
-	validators []parachaintypes.ValidatorID
+	validator          *validator
+	groups             map[parachaintypes.CoreIndex][]parachaintypes.ValidatorIndex
+	validators         []parachaintypes.ValidatorID
```

Comment:
What is the diff between ValidatorID and ValidatorIndex?

Reply:
ValidatorID is public key and ValidatorIndex is uint32 type which is lightweight replacement of the `ValidatorId`

Reply:
I have raised an issue to rename the type https://github.com/ChainSafe/gossamer/issues/4552